docs/malformed_files.md

```diff
@@ -0,0 +1,72 @@
+# Dealing with malformed files
+
+## Field compatibility
+
+When loading multiple files, the `INFO` and `FORMAT` fields
+from all VCF files are merged together to generate a "representative header",
+which is then used to generate a single BigQuery schema. If the same key is
+defined in multiple files, then its definition must be compatible across files.
+The compatibility rules are as follows:
+
+* Fields are compatible if they have the same `Number` and `Type` fields.
+  Annotation fields (i.e. those specified by `--annotation_fields`) must also
+  have the same `Description`.
+
+* Fields with different `Type` values are compatible in the following cases:
+
+  * `Integer` and `Float` fields are compatible and are converted to `Float`.
+  * We are adding more compatibility options (e.g. resolving `String` and
```

Comment:
Please update. "We have added...." or add something like: "You must run the pipeline with --allow-incompatible-records to automatically resolve conflicts between incompatible fields (e.g. String and Integer). This is to ensure incompatible types are not silently ignored.".

Reply:
Done